### PR TITLE
Make DefaultCookieName a static readonly field instead of a property

### PR DESCRIPTION
**Pull request by hishamco**
I made this PR based on issue #36

### CODE REVIEW

**Comment by kirthik** [comment]
The corresponding issue for this PR is not triaged. I will wait for the triage to make a decision before looking at the PR.

**Comment by Eilon** [comment]
I closed the issue - it's not something we should do.

**Comment by hishamco** [comment]
Check the my last comment in the issue #36

File: src/Microsoft.AspNet.Localization/CookieRequestCultureProvider.cs [hunk 1]
```diff
@@ -17,6 +17,10 @@ public class CookieRequestCultureProvider : RequestCultureProvider
         private static readonly char[] _cookieSeparator = new[] { '|' };
         private static readonly string _culturePrefix = "c=";
         private static readonly string _uiCulturePrefix = "uic=";
+        /// <summary>
```
Comment (Eilon) [inline src/Microsoft.AspNet.Localization/CookieRequestCultureProvider.cs]: Add an empty line here.

Comment (hishamco) [inline src/Microsoft.AspNet.Localization/CookieRequestCultureProvider.cs]: Do you mean after ```_uiCulturePrefix```? I already added what I suggest with the docs, i'm not sure if empty line is needed

Comment (Eilon) [inline src/Microsoft.AspNet.Localization/CookieRequestCultureProvider.cs]: It looks weird to have a doc comment "touching" the field above it.

**Comment by hishamco** [comment]
The PR has been updated
Thanks @Eilon

**Comment by kirthik** [comment]
Looks good to me

**Comment by Eilon** [comment]
@kirthik please merge.

**Comment by kirthik** [comment]
merged c228b65d478561869daa9161fc97a89ff02cad2e

**Pull request closed without merging**